src/test/java/org/jetbrains/nativecerts/mac/SecurityFrameworkUtilTest.java

```diff
@@ -87,6 +93,17 @@ public void addRealUserTrustedCertificate_deny() throws Exception {
         customUserTrustedCertificateTest("ssl", "deny", false);
     }
 
+    @Test
+    public void verifyCert() throws Exception {
+        List<X509Certificate> rootsAfter = SecurityFrameworkUtil.getTrustedRoots(SecurityFramework.SecTrustSettingsDomain.user);
+
+        List<String> aliases = rootsAfter.stream().map(crt -> crt.getSubjectDN().toString())
+                .collect(Collectors.toList());
+        assertThat(aliases, hasItem("CN=TIMJA-ROOT, O=TIMJA, ST=ES, C=UK"));
+        assertThat(aliases.size(), is(greaterThan(1)));
+        assertThat(aliases, hasItem("CN=TIMJA-INTERMEDIATE, O=TIMJA, ST=ES, C=UK"));
```

Comment:
just for testing locally, I'll look into generating on the fly once the code is working